wx/tls.h is needed by wx/string.h for UTF-8 builds, so move it from GUI_CMN_HDR
to BASE_CMN_HDR

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 build/bakefiles/files.bkl |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/build/bakefiles/files.bkl
===================================================================
--- a/build/bakefiles/files.bkl
+++ b/build/bakefiles/files.bkl
@@ -503,6 +503,7 @@ IMPORTANT: please read docs/tech/tn0016.
     wx/thread.h
     wx/thrimpl.cpp
     wx/timer.h
+    wx/tls.h
     wx/tokenzr.h
     wx/tracker.h
     wx/txtstrm.h
@@ -831,7 +832,6 @@ IMPORTANT: please read docs/tech/tn0016.
     wx/textctrl.h
     wx/textdlg.h
     wx/textentry.h
-    wx/tls.h
     wx/toolbar.h
     wx/validate.h
     wx/valtext.h
